components/authentication-framework/org.wso2.carbon.identity.application.authentication.framework/src/test/java/org/wso2/carbon/identity/application/authentication/framework/services/ConditionalAuthenticationMgtServiceTest.java

```diff
@@ -56,12 +56,13 @@ public void testGetAllAvailableFunctions(JsFunctionRegistry registry, String[] e
 
         JsFunctionRegistry emptyRegistry = new JsFunctionRegistryImpl();
         JsFunctionRegistry registryWithCustomFunctions = new JsFunctionRegistryImpl();
-        registryWithCustomFunctions.register(JsFunctionRegistry.Subsystem.SEQUENCE_HANDLER, "func1", new Object());
+        registryWithCustomFunctions.register(JsFunctionRegistry.Subsystem.SEQUENCE_HANDLER,
+                                             "func1", new Object());
 
         return new Object[][]{
-                {emptyRegistry, new String[]{"executeStep", "selectAcrFrom", "sendError", "Log.info"}},
+                {emptyRegistry, new String[]{"executeStep", "selectAcrFrom", "sendError", "Log.info", "require"}},
                 {registryWithCustomFunctions, new String[]{"executeStep", "selectAcrFrom", "sendError", "Log.info",
-                        "func1"}},
+                        "require", "func1"}},
```

Comment:
Can't we get this to a single line? Does it exceed 120?

Reply:
We cannot get this into a single line. It does exceed 120 characters.